CONTRIBUTING.md

```diff
@@ -158,19 +169,6 @@ Despite being named 'unit', these are integration tests, making sure public API
 npm run unit
 ```
 
-- To run tests in parallel, use `-j` flag:
```

Comment:
Is this not useful anymore?

Reply:
No, Mocha doesn't support this out of the box.